src/help/h_chairsguide.php

```diff
@@ -165,20 +162,12 @@ static function print_assignments(HelpRenderer $hth, $gj) {
 
     static function print_chair_conflicts(HelpRenderer $hth) {
         echo $hth->subhead("Chair conflicts");
-        echo "<p>Chairs and system administrators can access any information stored in the
-conference system, including reviewer identities for conflicted papers.
-It is easiest to simply accept such conflicts as a fact of life. Chairs
-who can’t handle conflicts fairly shouldn’t be chairs. However, HotCRP
-does offer other mechanisms for conflicted reviews.</p>
+        echo "<p>Chairs and system administrators can access any information stored in the program system, including reviewer identities for conflicted papers. It is easiest to simply accept such conflicts as a fact of life. Chairs who can’t handle conflicts fairly shouldn’t be chairs. However, HotCRP does offer other mechanisms for conflicted reviews.</p>
```

Comment:
Could you not change the line breaks but just the text? again easier for future merge.